examples/spacy/conda.yaml

```diff
@@ -0,0 +1,10 @@
+name: spacy-example
+channels:
+  - defaults
+  - anaconda
+dependencies:
+  - python==3.6
```

Comment:
`python=3.6`?

Reply:
It seems both `=` and `==` work.

Reply:
`conda env export` uses `=` for conda dependencies and `==` for pip dependencies.

```
$ conda env export
name: mlflow-dev-env
channels:
  - conda-forge
  - defaults
dependencies:
  - _py-xgboost-mutex=2.0=cpu_0
  - blas=1.0=mkl
  - ca-certificates=2019.11.28=hecc5488_0
  - catboost=0.20.1=py36_0
  ...
  - pip:
    - absl-py==0.8.1
    - adal==1.2.2
    - alabaster==0.7.12
    - alembic==1.2.1
    ...
```

Reply:
Would you like me to switch it to '=' ? I was taking inspiration from the examples/sklearn_logistic_regression where also == is used. I can change it if you guys prefer it that way.

Reply:
Sorry for the confusion. `==` is fine.